plugins/k8saudit/pkg/k8saudit/k8saudit.go

```diff
@@ -54,7 +54,7 @@ func (k *Plugin) Info() *plugins.Info {
 		Name:        pluginName,
 		Description: "Read Kubernetes Audit Events and monitor Kubernetes Clusters",
 		Contact:     "github.com/falcosecurity/plugins",
-		Version:     "0.7.0",
+		Version:     "0.7.1",
```

Comment:
```suggestion
		Version:     "0.8.0",
```

Since some lists have been renamed and we are introducing a backward incompatible change, we would have increased  [major number](https://github.com/falcosecurity/rules/blob/main/RELEASE.md#versioning-a-ruleset). However, per SemVer spec, incrementing the `minor` number is still acceptable when the major is `0`.